src/closer-core.coffee

```diff
@@ -700,6 +700,26 @@ core =
         core.group_$_by(core.identity, coll))
      )
 
+  'not_$_every_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.every? pred,coll
```

Comment:
well changed it but tests are failing at two points : when they expect the function to throw exception in case of wrong parity. what I think the reason is since we are passing pred, coll to `core.every_$QMARK_`, even if coll is not given, it is passed as an empty variable